### PR TITLE
Polymer 3 Conversion

### PR DESCRIPTION
**Pull request by dbatiste**

### CODE REVIEW

File: d2l-loading-spinner.js [hunk 1]
```diff
@@ -111,73 +120,74 @@
...
 				</g>
 			</svg>
 			<svg viewBox="0 0 42 42" class="d2l-loading-spinner-bg" focusable="false">
 				<g fill="none" fill-rule="evenodd" transform="translate(5 5)">
-					<circle stroke-width="0.5" cx="16" cy="16" r="16" class="d2l-loading-spinner-bg-stroke"/>
-					<circle stroke="none" fill="#FFF" cx="16" cy="16" r="16"/>
-					<circle stroke-width="2" cx="16" cy="16" r="11"/>
+					<circle stroke-width="0.5" cx="16" cy="16" r="16" class="d2l-loading-spinner-bg-stroke"></circle>
+					<circle stroke="none" fill="#FFF" cx="16" cy="16" r="16"></circle>
+					<circle stroke-width="2" cx="16" cy="16" r="11"></circle>
```
Comment (dlockhart) [inline d2l-loading-spinner.js]: Interesting that it "fixed" the SVG

File: package.json [hunk 2]
```diff
@@ -1,29 +1,47 @@
...
     "loading"
   ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/BrightspaceUI/loading-spinner.git"
+  },
+  "name": "d2l-loading-spinner",
+  "scripts": {
+    "test:lint": "npm run test:lint:wc && npm run test:lint:js",
+    "test:lint:js": "eslint *.html demo/*.html",
```
Comment (dlockhart) [inline package.json]: Looks like this will miss all the new `.js` files

File: d2l-loading-spinner.js (hunk 1 above)
Comment (dbatiste) [reply d2l-loading-spinner.js]: Yeah that's interesting - results seem to be ok (no visual diff).

File: package.json (hunk 2 above)
Comment (dbatiste) [reply package.json]: Will look into this.

Comment (svanherk) [reply package.json]: I think the script expected the old line would be `"test:lint:html": "eslint *.html demo/*.html",`

Comment (dbatiste) [reply package.json]: Haha no worries!  Thanks!

**Pull request closed without merging**